package.json

```diff
@@ -58,7 +58,7 @@
     "test:js": "mocha --require src/lib/test-setup.js --recursive \"src/**/*-test.js\"",
     "deploy": "gh-pages -x -d build/web -r \"https://$GH_TOKEN@github.com/mozilla/FirefoxColor.git\"",
     "release:base": "npm-run-all clean build:web sign && mv addon.xpi build/web && npm run deploy",
-    "release:dev": "cross-env ADDON_URL='https://mozilla.github.io/FirefoxColor/addon.xpi' SITE_URL='https://mozilla.github.io/FirefoxColor/' SITE_ID=github npm run release:base"
+    "release:dev": "cross-env ADDON_URL='https://addons-dev.allizom.org/en-US/firefox/addon/firefox-color/' SITE_URL='https://mozilla.github.io/FirefoxColor/' SITE_ID=github npm run release:base"
```

Comment:
hmm ..I might have to rethink this a bit if it's a problem that the lang is hardcoded here.

Reply:
You can remove `en-US/`, and AMO will automatically redirect elsewhere.

E.g. try visiting https://addons.mozilla.org/firefox/addon/firefox-color/